pkg/commands/proxy.go

```diff
@@ -198,7 +198,7 @@ func expandSSHTokens(tokenized string, host *config.Host, gateway *config.Host)
 }
 
 func prepareHostControlPath(host, gateway *config.Host) error {
-	if !config.BoolVal(host.ControlMasterMkdir) && ("none" == host.ControlPath) {
+	if !config.BoolVal(host.ControlMasterMkdir) && ("" == host.ControlPath) {
```

Comment:
Can you check both values?

`"none"` is a valid value that should have the same effect as `""`